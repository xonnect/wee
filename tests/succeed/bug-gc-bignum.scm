;; This triggers gc inside bignum operations.
;; Before bignum temp vars were GC roots, this caused all kinds of
;; trouble.
;; Registering them as roots fixed the problem.
(displayln (* 1 3))
(displayln (* 15 3))
(displayln (* 21435 2141241))
