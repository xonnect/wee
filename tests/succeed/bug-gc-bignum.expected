3
45
45897500835
